android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -277,6 +275,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     response.putString("uri", uri.toString());
+    response.putString("urlPath", realPath);
```

Comment:
```urlPath``` was just for debug, i don't think it's usefull to send it to react-native since we allready send the uri